api/src/main/java/openconsensus/trace/propagation/PropagationComponent.java

```diff
@@ -25,31 +25,28 @@
  */
 public abstract class PropagationComponent {
   /**
-   * Returns the {@link BinaryFormat} with the provided implementations. If no implementation is
-   * provided then no-op implementation will be used.
+   * Returns the {@link BinaryFormat} for this implementation.
    *
-   * @return the {@code BinaryFormat} implementation.
+   * <p>If no implementation is provided then no-op implementation will be used.
```

Comment:
hm... maybe we can provide W3C as a default. This would be more useful for situations when you don't need implementation while still can do some manipulations with headers if needed